src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
```

Comment:
Great job @Rana-Mudassir 🤝

- [ ] You may always remove commented code from your project as this is regarded as a bad practice.